### ISSUE TITLE
[question][features request] finer icon size adjustment

### ISSUE THREAD

**Issue by Yasen6275** [falzonv/discreet-launcher#281]
Not sure if it is posble, can we have more positions on icon size slider.

**Comment by falzonv** [comment]
Hello,

*(Sorry for the late answer, I am quite busy these weeks.)*

Not really, at least not in-between other positions which I guess is what you meant...

Initially this setting was added because when using the launcher on a tablet, icons looked very small (phone size). My first plan was to make a toggle "Normal / Large" but then I discovered that the launcher could also be used on a TV and thought that even bigger icons would be needed for that case.

So I ended up using a slider with 4 positions numbered from 3 to 6 where the position is multiplied by 12 to get the icon size in dpi (which are then converted in pixels according to device density):
=> Position 3 gives an icon of 36dp (not really useful, but it makes the slider look better when the default value is selected)
=> Position 4 gives an icon of 48dp (icon size in the Android stock launcher, and default value in Discreet Launcher)
=> Position 5 gives an icon of 60dp (looks better than the default size on a tablet)
=> Position 6 gives an icon of 72dp (alternative for even bigger screens)

This setting was not designed for ultra-customization but just for the launcher to be more usable on larger screens.
Maybe I should mention it in the setting caption, something like "Icon size (for tablets)" instead of just "Icon size"...

Best regards.
